MVC/models/articlesModels.js

```diff
@@ -0,0 +1,12 @@
+const db = require('../../db/connection');
+
+exports.selectArticleById = (article_id) => {
+    const sql = 'SELECT * FROM articles WHERE article_id = $1';
+    return db.query(sql, [article_id]).then(({rows}) => {
+        if (rows.length === 0) {
+            return Promise.reject({status: 404, msg: 'Article not found'});
+        } else {
+            return rows;
```

Comment:
Please make sure you are sending back an object, not the array containing the object here. How could you extract that object from this 'rows' array here?